llvm/lib/Transforms/Utils/CodeLayout.cpp

```diff
@@ -725,6 +734,7 @@ class ExtTSPImpl {
       return std::make_tuple(A1->Id, B1->Id) < std::make_tuple(A2->Id, B2->Id);
     };
 
+    double PrevScore = 1e9;
```

Comment:
This should use a sentinel value instead of a magic 1e9.

You can use `std::optional<double>` for an absent value.

Reply:
removed the variable, as it seems to confuse people while the benefit is modest